treeherder/perf/management/commands/compute_criteria_formulas.py

```diff
@@ -58,10 +56,51 @@ def add_arguments(self, parser):
             metavar='BUG_COOLDOWN',
         )
 
+        parser.add_argument(
+            '--multiprocessing',
+            '-mp',
+            action='store_true',
+            help='''Experimental! Whether to use a process pool instead of a thread pool''',
+        )
+        subparser = parser.add_subparsers(dest='individually')
+        individual_parser = subparser.add_parser(
+            'individually',
+            help='Compute perf sheriffing criteria for individual framework/suite combo',
+        )
+        individual_parser.add_argument('framework', action='store')
+        individual_parser.add_argument('suite', action='store')
+        parser.add_argument('--test', default=None)
+
     def handle(self, *args, **options):
+        if options.get('individually'):
+            return self._handle_individually(options)
+
+        quant_period = options['quantifying_period']
+        bug_cooldown = options['bug_cooldown']
+        multiprocessed = options['multiprocessing']
+
+        init_params = (None, quant_period, bug_cooldown)
+        formula_map = {
+            'EngineerTraction': EngineerTractionFormula(*init_params),
+            'FixRatio': FixRatioFormula(*init_params),
+        }
+
+        tracker = CriteriaTracker(formula_map, multiprocessed=multiprocessed)
+        tracker.load_records()
+        start = time.time()
+        tracker.update_records()
+        duration = time.time() - start
+
+        print(f'{self.INITIAL_PROMPT_MSG}', end='')
+
+        for record in tracker:
+            print(record)
+        print(f"Took {duration:.1f} seconds")
+
+    def _handle_individually(self, options):
         framework = options['framework']
         suite = options['suite']
-        test = options['test']
+        test = options['suite']
```

Comment:
Could you please tell me why is it okay to use `options['suite']` for `test`?

Reply:
Good catch!